rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section11/rule_11_6.json

```diff
@@ -0,0 +1,142 @@
+        "Section": 11,
+        "Rule": 6,
+        "Test": "a",
+        "test_description": "A one story building, single zone building has service hot water piping. The service hot water piping losses are correctly modeled.",
+        "expected_rule_outcome": "pass",
+        "standard": {
+            "rule_id": "11-6",
+            "ruleset_reference": "Table G3.1 #11, baseline column, i",
+            "rule_description": "Piping losses shall not be modeled.",
+            "applicable_rmr": "Baseline Model",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "mandatory_rule": "No",
+            "schema_version": "0.0.34"
+        },
+        "rmd_transformations": {
+            "baseline": {
+                "id": "ASHRAE229 1",
+                "calendar": {
+                    "is_leap_year": false
+                },
+                "data_timestamp": "2024-02-12T12:00Z",
+                "ruleset_model_descriptions": [
+                    {
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "floor_name": "Floor1",
+                                                "spaces": [
+                                                    {
+                                                        "id": "Space 1",
+                                                        "service_water_heating_uses": [
+                                                            {
+                                                                "id": "SHW 1",
+                                                                "served_by_distribution_system": "SHW Distribution 1"
+                                                            }
+                                                        ]
+                                                    }
+                                                ]
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ],
+                        "service_water_heating_distribution_systems": [
+                            {
+                                "id": "SHW Distribution 1",
+                                "service_water_piping": [
+                                    {
+                                        "id": "SHW Piping 1",
+                                        "are_thermal_losses_modeled": true
```

Comment:
Can you flip the case (a) and (b) logic? This is because False is ```pass``` and True is ```fail```.

Reply:
bah! Yes, I got them mixed up. Thanks Yun Joon. I will add that in my next push.